packages/core/test/acceptance/styling_spec.ts

```diff
@@ -2627,25 +2624,383 @@ describe('styling', () => {
     expect(getComputedStyle(div).width).toBe('10px');
   });
 
-  it('should allow classes with trailing and leading spaces in [ngClass]', () => {
+  it('should allow multiple styling bindings to work alongside property/attribute bindings', () => {
```

Comment:
This test fails in VE with `Invalid CSS style: unsafe` (but works in ivy) which tells me that there is a difference wrt to sanitisation.